### DISCUSSION TITLE
Welcome to likecoin-app Discussions!

### DISCUSSION THREAD

**Discussion by nwingt** [likecoin/likecoin-app#252] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by abhishekjmd** [comment]
Hii is the application / project published on playstore ??

**Reply by williamchong** [reply]
Yes it is https://liker.land/en/getapp

**Reply by abhishekjmd** [reply]
Got it
What's build-in command and environment setup file

**Reply by williamchong** [reply]
Not sure what do you mean, what do you want to do?

**Reply by abhishekjmd** [reply]
Hi williamchong,

Thanks for reaching out. I apologize if my previous message was unclear. I'm interested in setting up the project locally for learning purposes. I want to understand the codebase and contribute to the project if possible. Could you provide me with some guidance on how to get started? I appreciate any help or resources you can share.

Looking forward to your response.

**Reply by williamchong** [reply]
This is a standard react-native project init-ed by react-native cli, so following https://reactnative.dev/docs/environment-setup `React Native CLI Quickstart` would get you running
